ern-container-gen-android/src/AndroidGenerator.ts

```diff
@@ -34,9 +36,9 @@ import semver from 'semver'
 const PATH_TO_TEMPLATES_DIR = path.join(__dirname, 'templates')
 const PATH_TO_HULL_DIR = path.join(__dirname, 'hull')
 
-const DEFAULT_JSC_VARIANT = 'android-jsc'
-const DEFAULT_JSC_VERSION = '245459'
-const DEFAULT_HERMES_VERSION = '0.2.1'
+export const DEFAULT_JSC_VARIANT = 'android-jsc'
+export const DEFAULT_JSC_VERSION = '245459'
+export const DEFAULT_HERMES_VERSION = '0.2.1'
```

Comment:
can we please move these constants to `electrode-native/ern-core/src/android.ts`

Reply:
:+1: